playbooks/group_vars/all.yml

```diff
@@ -0,0 +1,7 @@
+ats_mtls_capable: >
+  {{
+    acs.version is version('7.4', 'ge') and
+    transform.version is version('3.1', 'ge') and
+    trouter.version is version('2.1', 'ge') and
+    acs_play_sfs_version is version('2.1', 'ge')
+  }}
```

Comment:
this is entirely based on current versions, so why don't put ats_mtls_capable=true for 7.4 and greater and false for 7.3 in the acs specific vars?